pkg/network/tracer/tracer_linux_test.go

```diff
@@ -71,9 +71,9 @@ func doDNSQuery(t *testing.T, domain string, serverIP string) (*net.UDPAddr, *ne
 	dnsClient := new(dns.Client)
 	dnsConn, err := dnsClient.Dial(dnsServerAddr.String())
 	require.NoError(t, err)
-	defer dnsConn.Close()
 	dnsClientAddr := dnsConn.LocalAddr().(*net.UDPAddr)
 	_, _, err = dnsClient.ExchangeWithConn(queryMsg, dnsConn)
+	_ = dnsConn.Close()
```

Comment:
why this change?

Reply:
This suppresses an IDE complaint that we're not dealing with the error.. the other option is to wrap this in a closure which I feel is overkill for this use case. `defer` is nice, but we also are closing this connection very close to when we open it anyway